webapp/store/views.py

```diff
@@ -306,6 +306,38 @@ def publisher_details(publisher):
         if not context:
             flask.abort(404)
 
+        if "publishers" in context:
+            context["snaps"] = []
+            for publisher in context["publishers"]:
+                try:
+                    searched_results = api.get_searched_snaps(
+                        "publisher:" + publisher, size=500, page=1
+                    )
+                except ApiError as api_error:
+                    status_code, error_info = _handle_errors(api_error)
+
+                snaps_results = logic.get_searched_snaps(searched_results)
+                context["snaps"] = context["snaps"] + [
```

Comment:
Does `context["snaps"].extend([snap for snap in snaps_results if snap["apps"]])` work?